src/sentry/testutils/cases.py

```diff
@@ -1193,7 +1195,10 @@ def setUp(self):
 
         self.init_path = reverse(
             "sentry-organization-integrations-setup",
-            kwargs={"organization_slug": self.organization.slug, "provider_id": self.provider.key},
+            kwargs={
+                "organization_slug": self.organization.slug,
```

Comment:
Is this missing the change?

Reply:
it auto formatted, i haven't updated this endpoint yet.